### ISSUE TITLE
Found multiple specifications for AirshipExtensions

### ISSUE THREAD

**Issue by flexaddicted** [urbanairship/urbanairship-cordova#364]
❗For how-to inquiries involving Airship functionality or use cases, please
contact (support)[https://support.airship.com/].

# Preliminary Info

### What Airship dependencies are you using?

urbanairship-cordova: 12.0.0

### What are the versions of any relevant development tools you are using?

Xcode: 12.0.1
CocoaPods: 1.9.3

# Report

### What unexpected behavior are you seeing?

Running `pod install`, CocoaPods prints the following output:

```
[!] Found multiple specifications for `AirshipExtensions (14.1.1)`
```

### What is the expected behavior?

`pod install` should not prints the above warnings

### What are the steps to reproduce the unexpected behavior?

Setup a Cordova projects and build the iOS app.

Th problem is due to the fact Airship Cordova plugin uses a wrong spec source.

It should be source `https://cdn.cocoapods.org/` instead of `https://github.com/CocoaPods/Specs.git`.

### Do you have logging for the issue?

```
[!] Found multiple specifications for `AirshipExtensions (14.1.1)`:
- /Users/USER/.cocoapods/repos/trunk/Specs/d/c/5/AirshipExtensions/14.1.1/AirshipExtensions.podspec.json
- /Users/USER/.cocoapods/repos/master/Specs/d/c/5/AirshipExtensions/14.1.1/AirshipExtensions.podspec.json

[!] Found multiple specifications for `AirshipExtensions (14.1.0)`:
- /Users/USER/.cocoapods/repos/trunk/Specs/d/c/5/AirshipExtensions/14.1.0/AirshipExtensions.podspec.json
- /Users/USER/.cocoapods/repos/master/Specs/d/c/5/AirshipExtensions/14.1.0/AirshipExtensions.podspec.json

[!] Found multiple specifications for `AirshipExtensions (14.1.0-beta1)`:
- /Users/USER/.cocoapods/repos/trunk/Specs/d/c/5/AirshipExtensions/14.1.0-beta1/AirshipExtensions.podspec.json
- /Users/USER/.cocoapods/repos/master/Specs/d/c/5/AirshipExtensions/14.1.0-beta1/AirshipExtensions.podspec.json

...
```

**Comment by rlepinski** [comment]
I dont think this is an error with our stuff, but rather a cocoapods issue - https://github.com/CocoaPods/CocoaPods/issues/2861. You can try patching your pod file by following these instructions - https://github.com/CocoaPods/CocoaPods/issues/2861#issuecomment-584137389

**Comment by flexaddicted** [comment]
@rlepinski Thanks for your reply. Do you think executing the following command can fix the issue? The new Cordova iOS version uses the CDN, not the GitHub url.
I will end with the same problem, I guess.
 
```suggestion
pod repo remove master
```

**Comment by rlepinski** [comment]
Ahh - https://github.com/urbanairship/urbanairship-cordova/blob/main/plugin.xml#L193

We will take a look and try to fix it in the next release. I think that command will fix it until you regenerate the project, or maybe you can modify the podfile directly to remove the other source.

**Comment by rlepinski** [comment]
@flexaddicted It should be fixed in 12.0.1 🤞 , we are not using the default cdn

**Comment by flexaddicted** [comment]
Thanks @rlepinski
